### PR TITLE
[usage] Filter out not started workspace instances

### PR DESCRIPTION
**Pull request by easyCZ**
## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-preview

### CODE REVIEW

File: components/usage/pkg/db/workspace_instance.go [hunk 1]
```diff
@@ -224,10 +223,8 @@ type WorkspaceInstanceForUsage struct {
 	Type               WorkspaceType  `gorm:"column:workspaceType;type:char;size:16;default:regular;" json:"workspaceType"`
 	UsageAttributionID AttributionID  `gorm:"column:usageAttributionId;type:varchar;size:60;" json:"usageAttributionId"`
 
-	CreationTime VarcharTime `gorm:"column:creationTime;type:varchar;size:255;" json:"creationTime"`
```
Comment (easyCZ) [inline components/usage/pkg/db/workspace_instance.go:227]: Removes these fields as they are just confusing. They did not get removed originally when we switched to `stoppingTime`

File: components/usage/pkg/db/workspace_instance.go [hunk 2]
```diff
@@ -134,6 +134,7 @@ func ListWorkspaceInstancesInRange(ctx context.Context, conn *gorm.DB, from, to
 		Where(
 			conn.Where("wsi.stoppingTime >= ?", TimeToISO8601(from)).Or("wsi.stoppingTime = ?", ""),
 		).
+		Where("wsi.startedTime != ?", "").
```
Comment (geropl) [inline components/usage/pkg/db/workspace_instance.go:137]: :dart:

**Review by geropl — APPROVED** [review]
Thansk @easyCZ ! :rocket:

File: components/usage/pkg/db/workspace_instance.go (hunk 2 above)
Comment (easyCZ) [inline components/usage/pkg/db/workspace_instance.go:137]: This is where we filter out any without a started time

**Pull request merged**